distrobuilder/main_lxd.go

```diff
@@ -41,6 +42,7 @@ func (c *cmdLXD) commandBuild() *cobra.Command {
 	c.cmdBuild.Flags().StringVar(&c.flagType, "type", "split", "Type of tarball to create"+"``")
 	c.cmdBuild.Flags().StringVar(&c.flagCompression, "compression", "xz", "Type of compression to use"+"``")
 	c.cmdBuild.Flags().BoolVar(&c.flagVM, "vm", false, "Create a qcow2 image for VMs"+"``")
+	c.cmdBuild.Flags().UintVar(&c.flagImageSize, "size", 2147483648, "Size of the VM image"+"``")
```

Comment:
Any reason why we wouldn't have this in the yaml instead?
Could be a key under targets/lxd